src/components/avatar/avatar.tsx

```diff
@@ -1,19 +1,40 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { StyleProp, ViewStyle } from 'react-native';
 import { Avatar as AvatarUI } from 'react-native-paper';
 
 type AvatarProps = {
   size: number;
-  url?: string;
+  user?: null | {
+    firstName: string;
+    lastName: string;
+    avatar?: string;
+  };
   style?: StyleProp<ViewStyle>;
 };
 
-const Avatar: React.FC<AvatarProps> = ({ size, url, style }) => {
-  if (url) {
-    return <AvatarUI.Image size={size} source={{ uri: url }} style={style} />;
+const Avatar: React.FC<AvatarProps> = ({ size, user, style }) => {
+  const initials = useMemo<string>(() => {
+    if (!user) {
+      return '';
+    }
+
+    return `${user.firstName[0]}.${user.lastName[0]}.`;
+  }, [user]);
+
+  if (user?.avatar) {
+    return (
+      <AvatarUI.Image source={{ uri: user.avatar }} size={size} style={style} />
+    );
   }
 
-  return <AvatarUI.Text size={size} label="" style={style} />;
+  return (
+    <AvatarUI.Text
+      label={initials}
+      labelStyle={{ fontSize: size / 4 }}
```

Comment:
What is `4`? And why don't we just set `fontSize` explicitly?

Reply:
Because avatars can have different size.